doc/developer-notes.md

```diff
@@ -728,14 +728,20 @@ General Bitcoin Core
   - *Rationale*: RPC allows for better automatic testing. The test suite for
     the GUI is very limited.
 
-- Make sure pull requests pass CI before merging.
+- Make sure pull request has verified build and test compatibility across environments
```

Comment:
Prefer the current version of this line to the proposed change. It is clearer, shorter, and more grammatically correct.